backstory/src/main/webapp/perspective-demo.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Perspective</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato" />
+  </head>
+
+  <body class="vertical-flex-container" id="perspective-page">
+    <div class="content vertical-flex-container" id="perspective-content">
+      <h1 class="header blue-header">Check Toxicity</h1>
+
+      <form class="vertical-flex-container" id="perspective-form" onsubmit="printToxicity()">
```

Comment:
Does printToxicity() exist? What will happen when the user clicks on this?

If the function does not exist, it would be preferable to remove the reference to it, or remove this element altogether, until the function is added.

Reply:
It's not used yet, so I removed it.

Reply:
Done